src/test/java/com/google/javascript/clutz/goog_require_usage.ts

```diff
@@ -0,0 +1,10 @@
+const SimpleClass = goog.require('foo.SimpleClass');
+const {a} = goog.require('foo.simpleNamespace');
+
+// Asserting that types carry through.
+var b: number = a;
+// TODO(rado): this should be c: SimpleClass, but without ES6 module
```

Comment:
Oh that's a pretty big limitation, isn't it? `SimpleClass` cannot be used in a type position? Might be worth bringing that up with MS, I'd hope there's a way around that.